### PR TITLE
[py-pybedtools] added version 0.9.0

### PR DESCRIPTION
**Pull request by qwertos**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-pybedtools/package.py [hunk 1]
```diff
@@ -33,3 +34,6 @@ class PyPybedtools(PythonPackage):
     depends_on("py-pandas", type=("build", "run"))
     depends_on("py-pysam", type=("build", "run"))
     depends_on("py-six", type=("build", "run"))
+
+    def patch(self):
+        python('setup.py', 'cythonize')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-pybedtools/package.py]: Direct invocation of setup.py is [deprecated](https://blog.ganssle.io/articles/2021/10/setup-py-deprecated.html). Do the PyPI tarballs come pre-cythonized?

Comment (qwertos) [reply var/spack/repos/builtin/packages/py-pybedtools/package.py]: no

Comment (adamjstewart) [reply var/spack/repos/builtin/packages/py-pybedtools/package.py]: @pradyunsg I assume there's no other way to build this package without running `python setup.py cythonize` first?

Comment (qwertos) [reply var/spack/repos/builtin/packages/py-pybedtools/package.py]: Actually, I might have spoken too soon, let me double check something

Comment (qwertos) [reply var/spack/repos/builtin/packages/py-pybedtools/package.py]: So, the PyPI tarballs do come pre-cythonized, but the Github tarballs that this package was written with don't. How would you like me to handle this?

Comment (adamjstewart) [reply var/spack/repos/builtin/packages/py-pybedtools/package.py]: We should always prefer using the PyPI tarballs over the GitHub tarballs: https://spack.readthedocs.io/en/latest/build_systems/pythonpackage.html#pypi-vs-github

Can you switch from `url` to `pypi` and run `spack checksum` to recompute all the checksums?

Comment (qwertos) [reply var/spack/repos/builtin/packages/py-pybedtools/package.py]: Done

File: var/spack/repos/builtin/packages/py-pybedtools/package.py [hunk 2]
```diff
@@ -10,19 +10,18 @@ class PyPybedtools(PythonPackage):
...
+    version('0.7.10', sha256='518a2311bd33f29cf2ee8fc1a028dda8c8e380c9fc83fcb0fbaa206933174b50')
+    version('0.7.9',  sha256='e9134d7dc3de41b90126df633fe3aa854a0427d7ada834dc8681081e8b2fea14')
+    version('0.7.8',  sha256='e80e8b73b233ec6950069e0e1cf14127aff8310a8eaf3ee663bdfbcb61309fdc')
+    version('0.7.7',  sha256='dba8758bd86121e4dcfc8b25ebae7d3b2de6ff9e38e4cdd9dc03f128bf5d4c9d')
+    version('0.7.6',  sha256='8b6382036c715b49f83e35cc787d7084c1633ebe94a2a3c4102e805613f7426b')
+    version('0.7.5',  sha256='f2428b4845083eff36385bec241ddddf7488a7de3f18886a78c73226e9e3306c')
+    version('0.7.4',  sha256='15cfae9e8a207ded403ad9fa2e77f09d14c2fe377d1bc5f8b063647e2d0554e0')
+    version('0.6.9',  sha256='56915b3e2200c6fb56260a36f839e66ce27d7dd3ef55fba278c3931b786fbfd1')
 
     depends_on("py-setuptools", type="build")
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-pybedtools/package.py]:
```suggestion
    depends_on("py-setuptools@0.6c5:", type="build")
```

**Review by adamjstewart — DISMISSED** [review]

**Comment by adamjstewart** [comment]
@spackbot fix style

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**